hooks/pre/20-certs_update.rb

```diff
@@ -0,0 +1,66 @@
+require 'fileutils'
+require 'English'
+
+if module_enabled?('katello')
```

Comment:
Also, needs to work on fpc
```suggestion
if module_enabled?('certs')
```